tests/unit_tests/test_local_datastore.py

```diff
@@ -0,0 +1,10 @@
+"""Tests the LocalDataStore."""
```

Comment:
You should ideally not need to test local datastore IMO since it is only used for unit tests. If a method in there is not required for the unit tests I would say remove it's okay to remove that method.

Reply:
This file will be finally updated when we add more unit tests, right now I added this as dummy tests to have the builds pass.